packages/ui/src/components/molecules/PlanCard.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Check, CrossIcon } from '../../assets';
+import { useTheme } from '../../themes';
+import { Button, Flex, Typography } from '../atoms';
+import { WidthProps, width } from '../utils';
+
+export type PlanCardType = 'silver' | 'gold';
+
+export interface PlanCardProps extends WidthProps {
+  plantype: PlanCardType;
+  heading: string;
+  tagline: string;
+  description: string;
+  price: string;
+  duration: string;
+  buttonText: string;
+  popularTagText?: string;
+  features: { text: string; available: boolean }[];
+  handleClick: () => void;
```

Comment:
Use consistent prop names

```suggestion
  onClick: () => void;
```